packages/components/src/external-link/types.ts

```diff
@@ -0,0 +1,11 @@
+/**
+ * External dependencies
+ */
+import type { ReactNode } from 'react';
+
+export type ExternalLinkProps = {
+	/**
+	 * The content to be displayed within the link.
+	 */
+	children?: ReactNode;
```

Comment:
I feel like we should mark `children` as a required prop, no?

I'm also wondering if we should explicitly mark `href` as a required prop. What do you think? (cc @ciampo)

Reply:
👍 I agree. Will update later today if no objections.

Reply:
👍  Have marked them as required and synced the README file